### PR TITLE
Adopt v2.0 APIs

### PR DESCRIPTION
**Pull request by davidzhao**
WIP

### CODE REVIEW

**Comment by davidzhao** [comment]
@dennwc I'm trying this locally and ran into a few SIP related errors. Could you take a look on this branch?

```
cmd/livekit-cli/sip.go:353:24: sipClient.ListSIPParticipant undefined (type *lksdk.SIPClient has no field or method ListSIPParticipant)
cmd/livekit-cli/sip.go:353:74: undefined: livekit.ListSIPParticipantRequest
cmd/livekit-cli/sip.go:377:25: sipClient.DeleteSIPParticipant undefined (type *lksdk.SIPClient has no field or method DeleteSIPParticipant)
cmd/livekit-cli/sip.go:377:77: undefined: livekit.DeleteSIPParticipantRequest
cmd/livekit-cli/sip.go:389:44: info.SipParticipantId undefined (type *livekit.SIPParticipantInfo has no field or method SipParticipantId)
````

**Comment by dennwc** [comment]
@davidzhao You probably need to rebase. There's no list/delete for SIP Participant in the [latest CLI](https://github.com/livekit/livekit-cli/blob/main/cmd/livekit-cli/sip.go#L305).

**Comment by davidzhao** [comment]
> @davidzhao You probably need to rebase. There's no list/delete for SIP Participant in the [latest CLI](https://github.com/livekit/livekit-cli/blob/main/cmd/livekit-cli/sip.go#L305).

that worked, thanks!

File: cmd/livekit-cli/main.go [hunk 1]
```diff
@@ -52,9 +50,19 @@ func main() {
...
-	}
+		Before: func(c *cli.Context) error {
+			logConfig := &logger.Config{
+				Level: "info",
+			}
+			if c.Bool("verbose") {
+				logConfig.Level = "debug"
+			}
+			logger.InitFromConfig(logConfig, "livekit-cli")
+			lksdk.SetLogger(logger.GetLogger())
```
Comment (dennwc) [inline cmd/livekit-cli/main.go:61]: Maybe switch to v2 is a good moment to use `slog`? It seems like https://github.com/uber-go/zap/issues/1333 is not there yet, but other options like https://github.com/samber/slog-zap already exist.

Comment (davidzhao) [reply cmd/livekit-cli/main.go:61]: we can wait till things stabilizes a bit before updating to it

**Pull request merged**